app/src/main/kotlin/com/saltedge/authenticator/features/connections/list/ConnectionsListViewModel.kt

```diff
@@ -149,6 +153,18 @@ class ConnectionsListViewModel(
         }
     }
 
+    fun onViewConsentsOptionSelected() {
+        val index = onListItemClickEvent.value?.peekContent() ?: return
+        val connectionGuid = listItemsValues.getOrNull(index)?.guid ?: return
+
+        val listOfConsents = arrayListOf(consents[listItemsValues.getOrNull(index)?.connectionId])
```

Comment:
try to get a value of listItemsValues (`listItemsValues.getOrNull(index)`) just one time